src/qt/sendcoinsdialog.cpp

```diff
@@ -369,6 +369,8 @@ void SendCoinsDialog::on_sendButton_clicked()
         accept();
         CoinControlDialog::coinControl()->UnSelectAll();
         coinControlUpdateLabels();
+        const uint256 txid = currentTransaction.getTransaction()->GetHash();
```

Comment:
Avoid copy here:
```cpp
const uint256& txid = ...
```

Reply:
Or roll it into the Q_EMIT, as txid is not used after this. YMMV. For performance I'd be surprised it makes any difference.
```
Q_EMIT coinsSent(currentTransaction.getTransaction()->GetHash());
```

Reply:
Rolled into `Q_EMIT`.